daml-lf/encoder/src/main/scala/com/digitalasset/daml/lf/archive/testing/DamlLfEncoder.scala

```diff
@@ -29,7 +29,7 @@ private[daml] object DamlLfEncoder extends App {
     throw new Error("You should not get this error")
   }
 
-  private val pkgId = Ref.PackageId.assertFromString("-self-")
+  private val pkgId = Ref.PackageId.assertFromString("self")
```

Comment:
why do we need to change this one ?

Reply:
```suggestion
  private val pkgId = Ref.PackageId.assertFromString("-self-")
```

Reply:
leftover from when I tried to also restrict characters, dropped it